orchestra/migrations/0056_create_new_task_communication_type.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-04-29 06:44
+from __future__ import unicode_literals
+
+from enum import Enum
+
+from django.db import migrations
+
+# Copy pasted from models since migrations does not have access
+class CommunicationMethods(object):
+    SLACK = 'slack'
+    EMAIL = 'email'
+
+
+COMMUNICATION_METHODS = (
+    (CommunicationMethods.SLACK, 'Slack'),
+    (CommunicationMethods.EMAIL, 'Email'),
+)
+
+class CommunicationType(Enum):
+    TASK_STATUS_CHANGE = 0
+    NEW_TASK_AVAILABLE = 1
+
+def add_communication_type(apps, schema_editor):
+    Worker = apps.get_model('orchestra', 'Worker')
+    CommunicationPreference = apps.get_model('orchestra',
+                                             'CommunicationPreference')
+    communication_type = (
+        CommunicationType.NEW_TASK_AVAILABLE.value)
+    for worker in Worker.objects.all():
+        communication_preference, created = (
+            CommunicationPreference.objects.get_or_create(
+                worker=worker,
+                communication_type=communication_type))
+
+        # By default set both Slack and Email notifications to True
+        if created:
+            communication_preference.methods.slack = True
```

Comment:
we should test manually but i don't think this works. I think you need to say `communication_preference.methods.slack  = CommiunicationPreference.methods.slack`